collector/collector.go

```diff
@@ -98,8 +98,8 @@ func newLabelGauge(system, subsystem, name, help, prefix, label string) *prometh
 		Help:      help,
 	}
 	metric := prometheus.NewGaugeVec(opts, []string{"server_id", label})
-
-	Tracef("Created metric: %s, %s, %s, %s", namespace, subsystem, name, help)
+	metric.
```

Comment:
This intentional?

Reply:
Nope! sorry, I think I just hit enter in the wrong place :sweat_smile: